GoogleAnalyticsEventForwarder.js

```diff
@@ -25,7 +25,8 @@
             OptOut: 6,
             Commerce: 16
         },
-        trackerCount = 1;
+        trackerCount = 1,
+        nonInteractionFlag = 'Google.nonInteraction';
```

Comment:
should we make this a public constant? if so, perhaps it should have a name what makes it look like a constant? NON_INTERACTION_FLAG or something.

Also this is minor, but how about Google.NonInteraction, to make the casing in line with other custom flags that we use. Speaking of which..maybe we should support them in JS?

http://docs.mparticle.com/integrations/google-analytics/event/#event-tracking

Reply:
hm

i took a look through any other times `customFlags` were used in other forwarders and it looks like there aren't any public constants, but rather the integration docs spell out how they should be used

* [bing](https://github.com/mparticle-integrations/mparticle-javascript-integration-bingads/blob/master/BingAdsEventForwarder.js#L168)
* [criteo](https://github.com/mparticle-integrations/mparticle-javascript-integration-criteo/blob/master/CriteoEventForwarder.js#L97)
* [localytics](https://github.com/mparticle-integrations/mparticle-javascript-integration-localytics/blob/master/LocalyticsEventForwarder.js#L150)
* [simplereach](https://github.com/mparticle-integrations/mparticle-javascript-integration-simplereach/blob/master/SimpleReach.js#L18-L25)

i propose not doing public constants. mainly because the only reason developers will know to use the constant is to look at our docs, where it already explains to add `customFlags` in a certain way

I had originally done Google.nonInteraction because `nonInteraction` is how clients would otherwise enter it into GA methods directly, but good point. I'll change. Looks like we do have label, category, and value in the logEvent function in the GA forwarder, but I'll make them constants as well

Reply:
good for another 👀